dispatcher/producers/scheduled.py

```diff
@@ -24,13 +25,16 @@ async def start_producing(self, dispatcher) -> None:
     def all_tasks(self) -> list[asyncio.Task]:
         return self.scheduled_tasks
 
-    async def run_schedule_forever(self, task_name: str, per_seconds, dispatcher) -> None:
+    async def run_schedule_forever(self, task_name: str, per_seconds, dispatcher, submission_options) -> None:
         logger.info(f"Starting task runner for {task_name} with interval {per_seconds} seconds")
         while True:
             await asyncio.sleep(per_seconds)
             logger.debug(f"Produced scheduled task: {task_name}")
             self.produced_count += 1
-            await dispatcher.process_message({'task': task_name, 'uuid': f'sch-{self.produced_count}'})
+            message = submission_options.copy()
+            message['task'] = task_name
+            message['uuid'] = f'sch-{self.produced_count}'
```

Comment:
Is not weird to use `uuid` for something that is not actually an uuid? 
https://www.rfc-editor.org/rfc/rfc9562.html
Would you consider a new issue to eventually redesign the schema of the message?

Reply:
> Would you consider a new issue to eventually redesign the schema of the message?

Yes. At this point in code we can introduce new fields without risking the 8,000 char limit. The presumed direction this is going in is that we would add additional fields to identify the origination of the message to replace what we were getting from `uuid`, which is log quality.

Some people might disagree, but I would rather keep `uuid` out of logs if it is a "real" UUID. One case for https://github.com/ansible/dispatcher/issues/8 is to pass logging identifiers that are bound from the original request or job. This requires a custom formatter on the app side. For out-of-the-box dispatcher, there's no reason to believe that a uuid is any more helpful for log tracing than origin information.